percona_pg_telemetry.c

```diff
@@ -780,12 +826,153 @@ get_extensions_list(PTDatabaseInfo *dbinfo, MemoryContext cxt)
 	return extlist;
 }
 
+static List *
+get_access_methods_list(MemoryContext cxt)
+{
+	List	   *amlist = NIL;
+	SPITupleTable *tuptable;
+	int			spi_result;
+	char	   *query = "SELECT am.amname AS access_method, count(*) FROM pg_class c LEFT JOIN pg_am am ON c.relam = am.oid WHERE c.relkind = 'r' AND c.relnamespace NOT IN ( SELECT oid FROM pg_namespace WHERE nspname IN ('pg_catalog', 'information_schema', 'pg_toast') ) GROUP BY access_method ORDER BY access_method;";
```

Comment:
Have you considered formatting these queries on multiple lines? I think that would improve readability.

Reply:
You should probably add a const qualifier too even if it does not matter too much.